Pipfile.lock

```diff
@@ -0,0 +1,194 @@
+{
+    "_meta": {
+        "hash": {
+            "sha256": "925fb4fb7b4fb6227dfc9730e817d95a220e00b5510b170b1b8e58757f3e3919"
+        },
+        "host-environment-markers": {
+            "implementation_name": "cpython",
+            "implementation_version": "3.6.2",
+            "os_name": "posix",
+            "platform_machine": "x86_64",
+            "platform_python_implementation": "CPython",
+            "platform_release": "4.13.3-1-zen",
+            "platform_system": "Linux",
+            "platform_version": "#1 ZEN SMP PREEMPT Wed Sep 20 07:17:03 UTC 2017",
```

Comment:
Is it okay if you commit your personal environment info?

Reply:
FYI https://github.com/kennethreitz/pipenv/issues/753